src/dbnode/integration/large_tiles_test.go

```diff
@@ -134,6 +134,12 @@ func TestReadAggregateWrite(t *testing.T) {
 	}, time.Second*10))
 
 	log.Info("validating aggregated data")
+
+	flushState, err := testSetup.DB().FlushState(trgNs.ID(), 0, dpTimeStart)
```

Comment:
Maybe add comments to the ids of series that you write, specifying which shard they are targeting. This is important aspect of this test, and it would be very difficult to figure out on your own.